airsonic-main/src/test/java/org/airsonic/player/api/jukebox/AbstractAirsonicRestApiJukeboxIntTest.java

```diff
@@ -176,10 +177,7 @@ private ResultMatcher playListItem1isCorrect() {
             jsonPath("$.subsonic-response.jukeboxPlaylist.entry[0].suffix").value(mediaFile.getFormat()).match(result);
             jsonPath("$.subsonic-response.jukeboxPlaylist.entry[0].duration").value(mediaFile.getDurationSeconds()).match(result);
             jsonPath("$.subsonic-response.jukeboxPlaylist.entry[0].bitRate").value(mediaFile.getBitRate()).match(result);
-
-            // The path is absolute, we remove the folder source with replace the folder with nothing
-            // and the "/" they stay with substring(1), must be a method in mediaFile for method to obtain the relative path is same every where.
-            jsonPath("$.subsonic-response.jukeboxPlaylist.entry[0].path").value(mediaFile.getPath().replace(mediaFile.getFolder(), "").substring(1)).match(result);
+            jsonPath("$.subsonic-response.jukeboxPlaylist.entry[0].path").value(SubsonicRESTController.getRelativePath(mediaFile, settingsService)).match(result);
```

Comment:
This test is used to be not cross-platform as the deleted comment mentions.

This change should make it cross platform.

Reply:
This should be in a separate PR.

Reply:
On one note, yes, perhaps, but on another, my tests on multiple systems are failing due to this and I can't get this PR to pass locally on a few systems without this change. I don't want to remove a change that is necessary for the PR to pass. So I'm going to keep it in this PR.